### ISSUE TITLE
Checkbox/extra page for volunteers 

### ISSUE THREAD

**Issue by sql-aus-hh** [sqlcollaborative/DataSaturdays#50]
Also we will need a way for participants to make clear that they are willing to volunteer during an event. 

Just by a checkbox or an extra page to opt-in for one or more tasks.

**Comment by amtwo** [comment]
Assuming that #49 is resolved by leveraging EventBrite (and/or similar platform) as the registration platform, would this need be met by having a separate ticket type for Volunteering? 

Setting up a separate volunteering ticket type (or multiple volunteer "tickets" for different tasks: Room proctor, front desk, lunch, etc) could provide the data needed & a relatively seamless UX

**Comment by suxstellino** [comment]
@amtwo I've tried a test event. Did you mean the following?
![IMG_20201221_234844.jpg](https://user-images.githubusercontent.com/2568107/102829325-2740da00-43e7-11eb-95b9-df7d28fd8d3d.jpg)

**Comment by amtwo** [comment]
Yes, something like that!

**Comment by MichaelAdrianJohnson** [comment]
Eventbrite allows you to provide custom questions on registration which will allow you as attendees if they would like to volunteer and even how (Room monitor, setup etc.) that might be easier than using ticket types.

**Comment by suxstellino** [comment]
You're right. anyways, perhaps, it's not simple to extract the volunteers from all the subscriptions based on those questions?

**Comment by MichaelAdrianJohnson** [comment]
I don't think its too difficult. 
I never tried but I think all this can be automated using API's and things like volunteer lists extracted from there. 
~maybe that's part of the tooling that gets built by\for the community.

**Comment by suxstellino** [comment]
I was trying to think about something that don't involve any code, using the tool instead of writing custom business logic. That being said, we can do anything 😉

**Comment by MichaelAdrianJohnson** [comment]
you can just export the tickets along with the answers (Volunteer opt ins etc.) from the Eventbrite site no tools needed. 
We could just document the process and add it to the event guide.

**Comment by SQLDBAWithABeard** [comment]
Added volunter request data to allow email or link for requesting volunteers #149
